### PR TITLE
feat(forced-decision): Adding acceptance tests for forced-decision.

### PR DESCRIPTION
**Pull request by yasirfolio3**
## Summary
- Adding acceptance tests for forced-decision parameters in decide-api.

### CODE REVIEW

File: tests/acceptance/test_acceptance/test_decide.py [hunk 1]
```diff
@@ -78,6 +110,53 @@ def test_decide__feature(session_obj, flag_key, expected_response, expected_stat
...
+              "ENABLED_FLAGS_ONLY",
+              "INCLUDE_REASONS"
+          ],
+          "userAttributes": {"attr_1": "hola"},
+          "forcedDecisions": [
+            {
+              "flagKey": \"""" + forced_flag + """\",
+              """+ rule_key +"""
+              "variationKey": \"""" + forced_variation + """\"
+            }
```
Comment (Mat001) [inline tests/acceptance/test_acceptance/test_decide.py:146]: The use of triple quotes looks a little confusing. Is that the only way? What about single quotes?

File: tests/acceptance/test_acceptance/test_decide.py [hunk 2]
```diff
@@ -78,6 +110,53 @@ def test_decide__feature(session_obj, flag_key, expected_response, expected_stat
     resp.raise_for_status()
 
 
+@pytest.mark.parametrize(
+    "flag_key, expected_response, expected_status_code, forced_flag, forced_rule, forced_variation", [
+        ("feature_2", expected_forced_decision_without_rule_key, 200, "feature_2", None, "variation_1"),
+        ("feature_2", expected_forced_decision_with_rule_key, 200, "feature_2", "feature_2_test", "variation_2")
+    ],
+    ids=["variation_1", "16931381940"])
+def test_decide_with_forced_decision__feature(session_obj, flag_key, expected_response, expected_status_code, forced_flag, forced_rule, forced_variation):
```
Comment (Mat001) [inline tests/acceptance/test_acceptance/test_decide.py:119]: What about other forced decision api's? @jaeopt what do you think?

Comment (Mat001) [reply tests/acceptance/test_acceptance/test_decide.py:119]: Also, could we have cariation override for forced decisions in /v1/override endpoint? Like here: https://github.com/optimizely/agent/blob/yasir/forced-decision-acceptance/tests/acceptance/test_acceptance/test_overrides.py

File: tests/acceptance/test_acceptance/test_decide.py [hunk 3]
```diff
@@ -214,4 +293,4 @@ def test_decide_403(session_override_sdk_key):
         assert resp.json()['error'] == 'unable to fetch fresh datafile (consider ' \
                                        'rechecking SDK key), status code: 403 Forbidden'
 
-        resp.raise_for_status()
+        resp.raise_for_status()
```
Comment (Mat001) [inline tests/acceptance/test_acceptance/test_decide.py:296]: add a new empty line

**Review by Mat001 — CHANGES_REQUESTED** [review]

**Review by jaeopt — APPROVED** [review]
LGTM

**Review by Mat001 — APPROVED** [review]

File: tests/acceptance/test_acceptance/test_decide.py [hunk 4]
```diff
@@ -78,6 +106,51 @@ def test_decide__feature(session_obj, flag_key, expected_response, expected_stat
...
+        "userId": "matjaz",
+        "decideOptions": [
+            "ENABLED_FLAGS_ONLY",
+            "INCLUDE_REASONS"
+        ],
+        "userAttributes": {"attr_1": "hola"},
+        "forcedDecisions": [
+            {
+              "flagKey": forced_flag,
+              "ruleKey": "{}".format(forced_rule),
```
[the remark this answers is not in the document]
Comment (Mat001) [reply tests/acceptance/test_acceptance/test_decide.py:139]: @msohailhussain @yasirfolio3 Feel free to merge. String formatting would be nice to update, other than that lgtm.

**Pull request merged**